src/app/components/implementation-view/qpu-selection/qpu-selection.component.ts

```diff
@@ -735,19 +736,28 @@ export class QpuSelectionComponent implements OnInit, AfterViewInit {
   }
 
   saveResultsToQhana() : void {
-    const request = new XMLHttpRequest();
-    const form = new FormData();
-    
-    let results = JSON.stringify(this.analyzerResults);
-    form.append('results', results);
-    
     this.refreshNisqImpl();
-    request.addEventListener('load', (event) => {
-      this.qhanaService.notifyParentOnSaveResults(this.nisqImpl.fileLocation, event.currentTarget['responseURL'])
+    const body =JSON.stringify({
```

Comment:
Why is there now an extra wrapper around the json results?

The json will look like this:
```json
{
    "results": "{...}"
}
```
Where `results` is a string containing a json encoded value.
Either use the value now in `result` directly as the body or do not encode it as a string (only one `JSON.stringify` call should remain).